packages/file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h

```diff
@@ -3,4 +3,5 @@
 // found in the LICENSE file.
 
 #import <Foundation/Foundation.h>
-#import <file_selector_ios/FFSFileSelectorPlugin.h>
\ No newline at end of file
+
```

Comment:
Formatter didn't like this:
```
These files are not formatted correctly (see diff below):
  file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h

To fix run "pub global activate flutter_plugin_tools && pub global run flutter_plugin_tools format" or copy-paste this command into your terminal:
patch -p1 <<DONE
diff --git a/packages/file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h b/packages/file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h
index 122724e46..d79d3642b 100644
--- a/packages/file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h
+++ b/packages/file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h
@@ -3,5 +3,4 @@
 // found in the LICENSE file.
 
 #import <Foundation/Foundation.h>
-
 #import <file_selector_ios/FFSFileSelectorPlugin.h>

DONE
```

Reply:
Hm, clang-format must assume that any framework-style `#import` should be a single block. Oh well.